samples/bluetooth/nrf_dm/src/peer.c

```diff
@@ -134,7 +141,11 @@ static void led_notification(const struct peer_entry *peer)
 	if (peer->result.ranging_mode == DM_RANGING_MODE_RTT) {
 		res = peer->result.dist_estimates.rtt.rtt;
 	} else {
+#ifdef CONFIG_DM_USE_HIGH_PRECISION_CALC
+		res = peer->result.dist_estimates.mcpd.high_precision;
```

Comment:
the high precision estimate may be NaN, should we take the mcpd.best in that case?

Reply:
Yes, I think you're right.